src/main/typescript/node_modules/@atomist/rug/operations/MyHandlers.ts

```diff
@@ -1,103 +1,98 @@
 
 **/
 
 //NOTE: use case 1: Reopen an issue twice and handle success/failure
 
-
-export let simpleHandler: EventHandler<Issue,Issue> = {
-
-  expression: new ClosedIssues(),
-
-  name: "ClosedIssueReopener",
-  description: "Reopens closed issues",
-  tags: ["github", "issues"],
+@EventHandler("ClosedIssueReopener","Reopens closed issues", new ClosedIssues())
+@Tags("github", "issues")
+class SimpleHandler implements HandleEvent<Issue,Issue>{
   handle(match: Match<Issue,Issue>): Plan {
     let issue = match.root()
     let reopen = issue.reopen
-
-    reopen.onSuccess = {text: `Issue ${issue.number} has been reopened successfully`}
-    reopen.onError = {name: "sendFailureMessage", parameters: {issue: issue, who: "jess", blah: sendFailureMessage}}
-    return new Plan()
-      .add(reopen)
+  //  reopen.onSuccess = {text: `Issue ${issue.number} has been reopened successfully`}
+  //  reopen.onError = {name: "sendFailureMessage", parameters: {issue: issue, who: "jess"}}
+    return new Plan().add(reopen)
   }
 }
 
-// try to reopen the issue one more time!
-export function sendFailureMessage(error: Response<Issue>, p: any) : Plan {
-  let issue = error.body
-  return new Plan()
-    .add(new Message(`Issue ${issue.number} was not reopened, trying again`))
-    .add(p.repeat.onError(new Message(`Issue ${issue.number} was not reopened. Giving up!`)))
+export let simple = new SimpleHandler();
+
+@ResponseHandler("IssueClosedResponder", "Logs failed issue reopen attempts")
+class IssueReopenFailedResponder implements HandleResponse<Issue>{
+
+  @Parameter({description: "Name of recipient", pattern: "^.*$"})
+  who: string
+
+  handle(response: Response<Issue>): Plan {
+    let issue = response.body
+    let msg = new Message(`Issue ${issue.number} was not reopened, trying again`)
+    msg.channelId = this.who
+    return new Plan()
+      .add(msg)
+  }
 }
 
 //NOTE use case 2: run an editor/reviewer across a bunch of repos (the old Executor)
 
-class LicenseAdder implements CommandHandler {
-  name = "LicenseAdder"
-  description = "Runs the SetLicense editor on a bunch of my repos"
-  tags = ["github", "license"]
-  intent = ["add license"]
+@CommandHandler("LicenseAdder","Runs the SetLicense editor on a bunch of my repos")
+@Tags("github", "license")
+@Intent("add license")
+class LicenseAdder implements HandleCommand{
 
-  @parameter({description: "The name of the license", pattern: "^.*$"})
+  @Parameter({description: "The name of the license", pattern: "^.*$"})
   license: string;
 
   handle(command: CommandContext) : Plan {
-
     let result = new Plan()
-    // result.add(new Message("blah").addAction())
     var match: Match<TreeNode,Project> = command.pathExpressionEngine()
          .evaluate<TreeNode,Project>("/Team()/Owns::Org()/Has::Repo()")
-
     match.matches().forEach(project => {
       result.add({name: "blah", kind: "editor", project: project})
```

Comment:
Is the `name: "blah"` here the name of the editor to be run? In other words, if this were real code, it would be

```
result.add({name: "atomist-rugs:licensing-editors:AddLicenseFile", kind: "editor", project: project, parmeters: {license_name: license}})
```